tools/ci/tc/decision.py

```diff
@@ -121,24 +121,31 @@ def get_fetch_rev(event):
     is_pr, _ = get_triggers(event)
     if is_pr:
```

Comment:
Nit; could un-nest the below one level via:

```
if not is_pr:
  return (event["ref"], event["after"], None)
```

Reply:
Yes, but @Hexcles requested an assert which is easier if we have one return point so I'm going to leave this as-is.